WooCommerce/Classes/ViewRelated/Dashboard/MyStore/Cells/ProductTableViewCell.swift

```diff
@@ -61,10 +71,29 @@ extension ProductTableViewCell {
         )
         priceText = statsItem?.formattedTotalString
 
+        /// Set `center` contentMode to not distort the placeholder aspect ratio.
+        /// After a sucessfull image download set the contentMode to `scaleAspectFill`
+        productImage.contentMode = .center
         imageService.downloadAndCacheImageForImageView(productImage,
                                                        with: statsItem?.imageUrl,
                                                        placeholder: .productPlaceholderImage,
-                                                       progressBlock: nil,
-                                                       completion: nil)
+                                                       progressBlock: nil) { [productImage] (image, error) in
+                                                        guard image != nil, error == nil else { return }
+                                                        productImage?.contentMode = .scaleAspectFill
+        }
+    }
+}
+
+/// Constants
+///
+private extension ProductTableViewCell {
+    enum Constants {
+        static let cornerRadius = CGFloat(2.0)
+        static let borderWidth = CGFloat(0.5)
+    }
+
+    enum Colors {
+        static let imageBorderColor = UIColor.border
+        static let imageBackgroundColor = UIColor.systemColor(.systemGray6)
```

Comment:
I understand this is from `ProductsTabProductTableViewCell` but I think it's better to group these under **one specific umbrella**. The `Constants` and `Colors` names are not clear enough for me. They may as well be applied to `priceLabel`! 😄 

What do you think about this? 🙂 

```suggestion
    enum ProductImage {
        static let cornerRadius = CGFloat(2.0)
        static let borderWidth = CGFloat(0.5)
        static let borderColor = UIColor.border
        static let backgroundColor = UIColor.systemColor(.systemGray6)
    }
```

Reply:
It does express the intention better 👍